src/mainwindow.h

```diff
@@ -30,6 +30,11 @@
 #include <QMainWindow>
 #include <QScopedPointer>
 #include <QString>
+#include <QSortFilterProxyModel>
+#include <QTreeView>
+
+#include "callercalleemodel.h"
```

Comment:
foward-declare the model and the symbol, that should work - no need to include the fat headers here